app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class SessionsController < ApplicationController
+  before_action :check_admin?, only: [:new]
```

Comment:
cái này có sai ko cụ ơi, login thì đã có current user đâu

Reply:
cái này em muốn bắt là admin thì sẽ không cho vô lại page login nữa, phòng trường hợp người dùng gõ url trên browser. Em có dùng toán tử an toàn nên nếu chưa có current_user thì nó sẽ không check cái này ạ